src/components/com_tjcertificate/site/models/forms/externalcertificate.xml

```diff
@@ -0,0 +1,139 @@
+<?xml version="1.0" encoding="utf-8"?>
+<form>
+	<fieldset>
+		<field
+			name="id"
+			type="hidden"
+			default="0"
+			label="COM_TJCERTIFICATE_FORM_LBL_EVENT_ID"
+			readonly="true"
+			class="readonly"
+			description="JGLOBAL_FIELD_ID_DESC" />
+
+		<field
+			name="user_id"
+			type="hidden" />
+
+		<field
+			name="assigned_user_id"
+			type="sql"
+			label="COM_TJCERTIFICATE_FORM_LBL_ASSIGNED_USER"
+			description="COM_TJCERTIFICATE_FORM_LBL_ASSIGNED_USER_DESC"
+			query="SELECT id, name FROM #__users where block = 0"
+			key_field="id"
+			value_field="name"
+			>
+			<option value="">COM_TJCERTIFICATE_CERTIFICATE_USER_SELECT_OPTION</option>
+		</field>
+
+		<field
+			name="name"
+			type="text"
+			class="required"
+			label="COM_TJCERTIFICATE_FORM_LBL_CERTIFICATE_NAME"
+			description="COM_TJCERTIFICATE_FORM_LBL_CERTIFICATE_NAME_DESC"
+			required="true"
+			menu_published="0"
+			size="25"
+			filter=""
+			maxlength="250"/>
+
+		<field
+			name="unique_certificate_id"
+			type="text"
+			filter="safehtml"
+			label="COM_TJCERTIFICATE_CERTIFICATE_FORM_LBL_CERTIFICATE_ID"
+			description="COM_TJCERTIFICATE_CERTIFICATE_FORM_LBL_CERTIFICATE_ID_DESC"
+			maxlength="250"
+			size="25"
+		/>
+
+		<field
+			name="cert_url"
+			type="text"
+			label="COM_TJCERTIFICATE_FORM_LBL_CERTIFICATE_URL"
+			description="COM_TJCERTIFICATE_FORM_LBL_CERTIFICATE_URL_DESC"
+			menu_published="0"
+			size="25"
+			filter=""
+			maxlength="250"/>
+
+		<field
+			name="issuing_org"
+			type="text"
+			label="COM_TJCERTIFICATE_FORM_LBL_ISSUE_ORG"
+			description="COM_TJCERTIFICATE_FORM_LBL_ISSUE_ORG_DESC"
+			required="true"
+			menu_published="0"
+			size="25"
+			filter=""
+			maxlength="250"/>
+
+		<field
+			name="issued_on"
+			type="calendar"
+			default=""
+			required="true"
+			filter="user_utc"
+			format="%Y-%m-%d %H:%M:%S"
+			singleheader="true"
+			showtime="true"
+			timeformat="24"
+			todaybutton="false"
+			weeknumbers="false"
+			filltable="true"
+			minyear="-1"
```

Comment:
Remove this minyear="-1", if you set -1, the user can't got back more than 1 year, which is not the case here

Reply:
done